examples/job/create_and_submit_job.py

```diff
@@ -39,18 +46,17 @@
 material_endpoints = MaterialEndpoints(*ENDPOINT_ARGS)
 workflow_endpoints = WorkflowEndpoints(*ENDPOINT_ARGS)
 
-
 # Set job name.
 
 # In[]:
 
 
 JOB_NAME = "TEST JOB"
 
-
 # ## Retrieve IDs
 # 
-# Default account's materail and workflow are used in this example to create the job. Adjust the queries to use different material and workflow.
+# Default account's materail and workflow are used in this example to create the job. Adjust the queries to use different
+# material and workflow.
```

Comment:
Did something change in the formatting? Enforcing width?

Reply:
Yeah; used PyCharm to enforce linewidth in these

Reply:
Just to confirm: are we (1) changing the notebooks and then the python sources are updated accordingly, or (2) are we modifying the python sources directly after they were generated from the notebooks? It should be (1), otherwise when I open the notebook and modify it the python sources will be each time. To autoformat the notebooks we could use https://github.com/ryantam626/jupyterlab_code_formatter

Reply:
(1) is correct. Was generating the .py files automatically with jupyter's save hook, and then manually autoforamtting with pycharm.

New commits add in automated formatting of the generated .py files to eliminate this need.